json_serializable/build.yaml

```diff
@@ -2,8 +2,8 @@
 builders:
   json_serializable:
     target: ":json_serializable"
-    import: "package:json_serializable/builder.dart"
-    builder_factories: ["jsonSerializable"]
+    import: "package:json_serializable/json_serializable.dart"
+    builder_factories: ["createBuilderForJsonSerializable"]
```

Comment:
The changes in this file move away from the conventions we've been trying to establish. Perhaps I should document them somewhere...

Importing from `/builder.dart` is a signal that the code in here is meant more for the build system than for end users - it's good IMO to keep the builder separate from the annotations.

Having a noun name for the factory makes the build.yaml look a little nicer.

Reply:
Hrm...k